static/app/components/events/interfaces/performance/utils.tsx

```diff
@@ -1,42 +1,61 @@
   ),
   [IssueType.ERROR]: '',
 };
 
-const RESOURCE_LINKS: Record<IssueType, ResourceLink[]> = {
-  [IssueType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES]: [
-    {
-      text: t('Finding and Fixing Django N+1 Problems'),
-      link: 'https://blog.sentry.io/2020/09/14/finding-and-fixing-django-n-1-problems',
-    },
-    {
-      text: t('Rails Guide: Active Record Query Interface'),
-      link: 'https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations',
-    },
-    {
-      text: t('Django select_related and prefetch_related'),
-      link: 'https://betterprogramming.pub/django-select-related-and-prefetch-related-f23043fd635d',
-    },
-    // TODO: Update this when the blogpost has been written
-    // {
-    //   text: t('[Leave empty for future Sentry post]'),
-    //   link: 'https://sentry.io/',
-    // },
-  ],
-  [IssueType.ERROR]: [],
+type PlatformSpecificResources = Partial<Record<PlatformType, ResourceLink[]>>;
+
+// TODO: When the Sentry blogpost for N+1s and documentation has been released, add them as resources for all platforms
+const RESOURCE_LINKS: Record<IssueType, PlatformSpecificResources> = {
+  [IssueType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES]: {
+    python: [
+      {
+        text: t('Finding and Fixing Django N+1 Problems'),
+        link: 'https://blog.sentry.io/2020/09/14/finding-and-fixing-django-n-1-problems',
+      },
+      {
+        text: t('Django select_related and prefetch_related'),
+        link: 'https://betterprogramming.pub/django-select-related-and-prefetch-related-f23043fd635d',
+      },
+    ],
+    'python-django': [
+      {
+        text: t('Finding and Fixing Django N+1 Problems'),
+        link: 'https://blog.sentry.io/2020/09/14/finding-and-fixing-django-n-1-problems',
+      },
+      {
+        text: t('Django select_related and prefetch_related'),
+        link: 'https://betterprogramming.pub/django-select-related-and-prefetch-related-f23043fd635d',
+      },
+    ],
+    'ruby-rails': [
+      {
+        text: t('Rails Guide: Active Record Query Interface'),
+        link: 'https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations',
+      },
+    ],
+  },
+  [IssueType.ERROR]: {},
 };
 
 export function getResourceDescription(issueType: IssueType): string {
   return RESOURCES_DESCRIPTIONS[issueType];
 }
 
-export function getResourceLinks(issueType: IssueType) {
-  return RESOURCE_LINKS[issueType];
+export function getResourceLinks(
+  issueType: IssueType,
+  platform: PlatformType | undefined
+): ResourceLink[] {
+  if (!platform || !(platform in RESOURCE_LINKS[issueType])) {
+    return [];
+  }
+
+  return RESOURCE_LINKS[issueType][platform] as ResourceLink[];
```

Comment:
I have to assert as `ResourceLink[]` here otherwise TS thinks that it's possible that it will return `undefined`, even though it should always return an array due to the check above.

Reply:
I think TS will understand that this is defined if you change the line above to check:

`!RESOURCE_LINKS[issueType][platform]`

Reply:
Just gave it a shot but TS still doesn't seem understand :(
![image](https://user-images.githubusercontent.com/16740047/193046056-a86bbe72-2094-4d15-a670-6d2fe7bc6172.png)

Reply:
Damn, must be the double key that confuses TS. Oh well. You could also do `RESOURCE_LINKS[issueType][platform]!` or `RESOURCE_LINKS[issueType][platform] ?? []` but it doesn't matter too much

Reply:
true, I'll use nullish coalescence instead